### DISCUSSION TITLE
Get session from JWT token

### DISCUSSION THREAD

**Discussion by jeyjeycodes** [supabase/supabase#1782] [Questions]
Hi all,

is it possible to create a session from the JWT token? Preferably using the C# library but if it is possible in general then that would be good and I can work around it.

Kind regards,
Ahmed

**Comment by silentworks** [comment]
@jeylanicode I'm not sure what you mean by a session using the JWT token, in this instance, the JWT token acts as a session. Maybe rephrasing the question as to what you are trying to accomplish instead will help.

**Reply by jeyjeycodes** [reply]
For example, when we use the sign in method in C# it gives back a current session object/current user and etc. So when we make requests there is an active live session. Whereas if we just initialise with the the token those objects are no and there are no sessions. Sorry for the silly question. This is all quite new and confusing for me.

The scenario is. We get a jwt token via a mobile/web app which directly talks to supabase. Then we want to use that to talk to our backend which does a lot of custom and extra stuff. We want to then create a live session with that token and make the backend talk to our supabase database or auth stuff.
